app/assets/javascripts/static_pages.js

```diff
@@ -1 +1,219 @@
 
+// $(document).ready(function(){
```

Comment:
Pls merge after you did your thing
 On 17 Feb 2015 23:10, "Harry Chen" <notifications@github.com> wrote:

> In app/assets/javascripts/static_pages.js
> <https://github.com/cchchoi1986/hikehk/pull/23#discussion_r24821560>:
>
> > @@ -1 +1,219 @@
> >
> > +// $(document).ready(function(){
>
> We should remove this file
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/cchchoi1986/hikehk/pull/23/files#r24821560>.
>